frontend/src/components/layouts/layout.tsx

```diff
@@ -14,7 +14,7 @@ import LayoutWrapper from "./LayoutWrapper";
 const useStyles = makeStyles((theme) => ({
   mainHeading: {
     textAlign: "center",
-    margin: `${theme.spacing(4)}px 0`,
+    margin: `${theme.spacing(4)} 0`,
```

Comment:
is `px` automatically recognized in MUI5?

Reply:
Yes